openc3/lib/openc3/topics/limits_event_topic.rb

```diff
@@ -97,10 +97,12 @@ def self.current_set(scope:)
       sets(scope: scope).key('true') || "DEFAULT"
     end
 
-    def self.delete(_target_name, _packet_name, scope:)
+    def self.delete(target_name, packet_name, scope:)
       limits = Store.hgetall("#{scope}__current_limits")
       limits.each do |item, _limits_state|
-        Store.hdel("#{scope}__current_limits", item)
+        if item =~ /^#{target_name}__#{packet_name}__/
+          Store.hdel("#{scope}__current_limits", item)
+        end
```

Comment:
Previously we were just deleting everything? That's wrong so I delete based on matching the name.

Reply:
Good catch